### ISSUE TITLE
Validation Not Triggered

### ISSUE THREAD

**Issue by mikebronner** [dwightwatson/validating#43]
I'm not exactly sure what I'm doing wrong here, but for some reason I can't get it to validate as described in the documentation; it doesn't seem to trigger validation on save. Here is my code, using 0.8.4:

RolesController.php

``` php
use GeneaLabs\Role;

class RolesController extends \BaseController
{
    private $role;

    public function __construct(Role $role)
    {
        $this->role = $role;
    }

    public function store()
    {
        $role = $this->role->fill(Input::only('name', 'description'));
        $role->save();

        return Redirect::route('roles.index');
    }
}
```

BaseModel.php

``` php
namespace GeneaLabs;

use Watson\Validating\ValidatingModel;

class BaseModel extends ValidatingModel
{

}
```

Role.php

``` php
namespace GeneaLabs;

class Role extends BaseModel
{
    protected $rules = [
        'saving' => [
            'name' => 'required|min:3|unique:roles,name',
            'description' => 'required|min:10',
        ],
    ];

    protected $fillable = [
        'name',
        'description',
    ];

    public function users()
    {
        return $this->belongsToMany('GeneaLabs\User');
    }

    public function permissions()
    {
        return $this->belongsToMany('GeneaLabs\Permission');
    }
}
```

global.php:

``` php
App::error(function(ValidationException $exception, $code) {
    Flash::warning('Form validation failed');
    return Redirect::back()->withErrors($exception->getErrors())->withInput();
});
```

**Update:** did some more testing, and I can't get it to validate using ->isValid() either. Also tried using the trait directly on the model to see if that would make a difference, but no. :(
